### ISSUE TITLE
Maybe remove one level in the display tree

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#353]
Don't think we need the "Result Type (1 Error)" level in the display. Under Specimens (3 Errors), when it is open just have the list of error descriptions.  Two levels instead of three.
<img width="852" alt="screen shot 2018-06-11 at 7 23 39 pm" src="https://user-images.githubusercontent.com/1278789/41266698-01e70ff8-6dad-11e8-86b1-a6cb70c58300.png">

**Comment by rminnett** [comment]
I agree - the third level is intended for multiple validation errors for a single column, which could happen. When there's only one I can open it automatically.
